src/xturing/models/__init__.py

```diff
@@ -1,16 +1,22 @@
 from .base import BaseModel
 from .distilgpt2 import DistilGPT2, DistilGPT2LORA
-from .gpt2 import GPT2, GPT2LORA
-from .gptj import GPTJ, GPTJLORA
-from .llama import Llama, LlamaLORA
+from .gpt2 import GPT2, GPT2LORA, GPT2Int8, GPT2LORAInt8
+from .gptj import GPTJ, GPTJLORA, GPTJInt8, GPTJLORAInt8
+from .llama import Llama, LlamaInt8, LlamaLORA, LlamaLORAInt8
```

Comment:
Why is it LlamaLORA? I thought we had decided on something standard like GPT2Lora, GPTJLora, LlamaLora? Am I not correct? @MarcosRiveraMartinez @StochasticRomanAgeev